tests/kernel/install_ltp.pm

```diff
@@ -211,10 +211,16 @@ sub add_ltp_repo {
         }
 
         my $arch = '';
-        $arch = "_PowerPC"  if is_ppc64le();
-        $arch = "_zSystems" if is_s390x();
 
-        $arch = ((is_x86_64 || is_aarch64) ? "Tumbleweed" : "Factory") . $arch;
+        # ltp for leap15.2 is available only x86_64
+        # ltp for leap15.3+ is missing only s390x
+        if (is_leap('=15.2') && is_x86_64 || is_leap('15.3+') && !is_s390x) {
```

Comment:
IMO it can just be

```suggestion
        if (is_leap) {
```

because any test with an invalid combination of distro + arch would fail anyway. Currently all this achieves is that it adds the Tumbleweed repo instead.

Reply:
Sure, this could be done. From the test reviewer perspective, I would prefer to have `else { die ... }` if we hit invalid combination. WDYT ?

Reply:
Yep, it would make it more obvious that the test setup itself is invalid.

Reply:
```suggestion
        if ((is_leap('=15.2') && is_x86_64) || (is_leap('15.3+') && !is_s390x)) {
```